saleor/tests/e2e/orders/test_cannot_fullfill_order_with_invalid_shipping_method.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..channel.utils import create_channel
+from ..product.utils.preparing_product import prepare_product
+from ..shipping_zone.utils import (
+    create_shipping_method,
+    create_shipping_method_channel_listing,
+    create_shipping_zone,
+)
+from ..utils import assign_permissions
+from ..warehouse.utils import create_warehouse
+from .utils.draft_order_complete import raw_draft_order_complete
+from .utils.draft_order_create import draft_order_create
+from .utils.draft_order_update import draft_order_update
+from .utils.order_lines_create import order_lines_create
+
+first_address = DEFAULT_ADDRESS
+
+
+def prepare_shops(
+    e2e_staff_api_client,
+):
+    first_warehouse_data = create_warehouse(
+        e2e_staff_api_client,
+        name="Warehouse 1",
+        slug="warehouse-1",
+        address=first_address,
+    )
+    first_warehouse_id = first_warehouse_data["id"]
+
+    first_channel_data = create_channel(
+        e2e_staff_api_client, slug="channel-1", warehouse_ids=[first_warehouse_id]
+    )
+    first_channel_id = first_channel_data["id"]
+
+    first_shipping_zone_data = create_shipping_zone(
+        e2e_staff_api_client,
+        warehouse_ids=[first_warehouse_id],
+        channel_ids=[first_channel_id],
+    )
+    first_shipping_zone_id = first_shipping_zone_data["id"]
+
+    first_shipping_method_data = create_shipping_method(
+        e2e_staff_api_client, first_shipping_zone_id
+    )
+    first_shipping_method_id = first_shipping_method_data["id"]
+
+    create_shipping_method_channel_listing(
+        e2e_staff_api_client, first_shipping_method_id, first_channel_id
+    )
+
+    second_address = {
+        "firstName": "Jan",
+        "lastName": "Kowalski",
+        "phone": "+48123456787",
+        "companyName": "Saleor PL",
+        "country": "PL",
+        "countryArea": "",
+        "city": "WROCLAW",
+        "postalCode": "53-346",
+        "streetAddress1": "Smolna",
+        "streetAddress2": "13/1",
+    }
+    second_warehouse_data = create_warehouse(
```

Comment:
One warehouse is enough. You need to have two separate shipping methods for different countries.